src/config/env.ts

```
@@ -151,10 +151,10 @@ export const config = {
   logLevel: env.LOG_LEVEL,
   // Per-transport levels keep debug noise out of production aggregators (#398).
   logConsoleLevel:
-    env.LOG_LEVEL_CONSOLE ??
+    process.env.LOG_LEVEL_CONSOLE ??
     (env.NODE_ENV === "production" ? "info" : env.LOG_LEVEL),
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)